nixos/modules/services/misc/plikd.nix

```diff
@@ -0,0 +1,84 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.plikd;
+
+  format = pkgs.formats.toml {};
+
+  plikdCfg = format.generate "plikd.cfg" cfg.settings;
+in
+{
+  options = {
```

Comment:
It might be nice to provide some database related options and have the module automatically integrate with `postgresql`, if the sysadmin desires. Or not. 🤷‍♂️

Reply:
That is part of the problem with the tweaks I mention